src/setup-vcpkg/index.js

```diff
@@ -0,0 +1,158 @@
+const core = require('@actions/core');
+const tc = require('@actions/tool-cache');
+const exec = require('@actions/exec');
+const path = require('path');
+const fs = require('fs'); // Needed for verifySHA512 stream
+const crypto = require('crypto'); // Needed for verifySHA512
```

Comment:
suggest to rename to `require("node:<internal_module_name")` to make it clear it's not a 3rd party dependency.